### PR TITLE
Added Login Activity With Shared Preferences Class

### PR DESCRIPTION
**Pull request by aggarwalpulkit596**
Added Login Activity With Shared Preferences Class

### CODE REVIEW

**Comment by the-dagger** [comment]
@aggarwalpulkit596 From now onwards let's merge the PR after it has at least one approval from the contributors.

**Comment by aggarwalpulkit596** [comment]
@the-dagger okay bhaiya 👍

File: app/src/main/java/com/codingblocks/cbonlineapp/LoginActivity.kt [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import kotlinx.android.synthetic.main.activity_login.*
+import org.jetbrains.anko.AnkoLogger
+import org.jetbrains.anko.intentFor
+import org.jetbrains.anko.singleTop
+
+
+class LoginActivity : AppCompatActivity(), AnkoLogger {
+
+    private val CLIENT_ID = "9047417470"
+    private val REDIRECT_URI = "http://localhost"
```
Comment (championswimmer) [inline app/src/main/java/com/codingblocks/cbonlineapp/LoginActivity.kt]: redirect to something like https://app.codingblocks.com and whitelist that on the server. That is more secure. 
Also in manifest add that to scheme

Comment (aggarwalpulkit596) [reply app/src/main/java/com/codingblocks/cbonlineapp/LoginActivity.kt]: @championswimmer done in the next pr

**Pull request merged**